stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    kubernetes.io/cluster-service: "true"
+    release: {{ .Release.Name | quote }}
+  namespace: {{ .Values.namespace }}
+spec:
+  type: {{ .Values.serviceType }}
+  ports:
+  - name: http
+    port: 80
+    targetPort: http
```

Comment:
The named target port `http` doesn't exist in your deployment. This service won't work.